core/src/main/res/values-v26/styles.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="AntennaPod.TextView.Small" parent="@android:style/TextAppearance.Medium">
```

Comment:
Why do you specify one of the styles in `values-v26` and the others in `values-v28`?

Reply:
It gave me suggestion that it is required.